estate/models/estate_property_offer.py

```diff
@@ -1,4 +1,5 @@
-from odoo import fields, models
+from datetime import timedelta,datetime
```

Comment:
This should be sorted alphabetically, leave it as it is now because there is no sense this case

Just be aware if someone tells you this in the future (we know who 😆)